.env.example

```
@@ -72,4 +72,15 @@ ENCRYPTION_KEY=your_encryption_key_here_minimum_32_chars
 # Docker Hub (CI/CD용)
 # ============================================
-DOCKERHUB_USERNAME=your_dockerhub_username
\ No newline at end of file
+DOCKERHUB_USERNAME=your_dockerhub_username
+
+# ============================================
+# Monitoring (Grafana, Prometheus)
+# ============================================
+ENVIRONMENT=prod
+GRAFANA_ADMIN_USER=admin
+GRAFANA_ADMIN_PASSWORD=your_secure_password_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GRAFANA_ADMIN_PASSWORD key should go before the GRAFANA_ADMIN_USER key

(UnorderedKey)